echopype/convert/parsed_to_zarr_ek60.py

```diff
@@ -19,6 +19,23 @@ def __init__(self, parser_obj):
         self.p2z_ch_ids = {}  # channel ids for power, angle, complex
         self.datagram_df = None  # df created from zarr variables
 
+        # get the channel sort rule for EK60 type sensors
+        if "transceivers" in self.parser_obj.config_datagram:
+
+            # get channel and channel_id association and sort by channel_id
+            channels = list(self.parser_obj.config_datagram["transceivers"].keys())
+            channel_ids = {
+                ch: self.parser_obj.config_datagram["transceivers"][ch]["channel_id"]
+                for ch in channels
+            }
+            sorted_channel = dict(sorted(channel_ids.items(), key=lambda item: item[1]))
+
+            # get just the channel
+            sorted_channel = list(sorted_channel.keys())
+
+            # obtain sort rule for the channel index
+            self.channel_sort_rule = {str(ch): sorted_channel.index(ch) for ch in channels}
```

Comment:
I think you can do operation here with finding the sorted index in `channel_ids` and then find the position of the sorted index in `channels` without constructing `sorted_channel` and then only getting its keys, since the desired outcome here is the values being what can be used to index `channel` and the 0-based sequence of each channel.  

It is a little cryptic to construct `sorted_channel` and then only getting its keys and then use that for operation.

Reply:
After making this comment I see that you have basically that for the EK80 counterpart in `parsed_to_zarr_ek80.py`.

Reply:
I see, I will go ahead and implement what I did for EK80 here.